docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -5819,13 +5819,16 @@ System Version 0.9.1. Expected Version 1.0.0.
 The GRANT statement grants access privileges on an SQL object and its columns to specified users or roles.
 Privileges can be granted on the object, on one or more columns, or both.
 
+TIP: As the owner who creates the object has all privileges by default, there is no need to grant privileges to the owner.
+The owner could grant/revoke privileges to/from other users or roles for safety.

```

Comment:
Word smith suggestion "The user or role that creates the object becomes the object owner and has all privileges on the object.  There is no need to grant privileges to the owner.  The owner can grant. ...

Reply:
Thanks @robertamarton, your comment has been incorporated :smile: